example-workflows/remote-docker-image.yml

```diff
@@ -0,0 +1,63 @@
+---
+name: 'deploy'
+
+# yamllint disable-line rule:truthy
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish-docker-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cloning repo
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CR_PAT }}
+
+      - name: Build and push docker image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
+          push: true
+          platforms: linux/amd64
+          tags: ghcr.io/${{ github.repository }}:latest
```

Comment:
Might be worth using the commit sha vs `latest` here? Not sure about registry limits, but tagging as latest will not signal to docker to pull the latest image when Dokku performs the deployment.

Reply:
Generally i try to avoid using the `latest` tag, but i have some concerns about ghcr limits which is why i'm only using the 1 tag. But I think you're right about docker not pull the image if the tag hasn't changed. I want to do some tests on my repo to confirm and i'll then update this PR.

Reply:
Confirmed, docker will not pull the image. 

I was thinking we could use [docker-options](http://dokku.viewdocs.io/dokku/advanced-usage/docker-options/), eg `dokku docker-options:add app run "--pull=always"` but that only affects the image being run (eg `dokku/app`) and not the base images defined in the Dockerfile. 

Am gonna give this some more thought, but likely i'll change this PR to use commit hash for the image tag.

Reply:
Maybe --no-cache?

Reply:
Seems like that should work but haven't had any success with the following:

- `dokku config:set app DOKKU_DOCKERFILE_CACHE_BUILD=false` or
- `dokku docker-options:add app build "--no-cache"`

..but it's getting pretty late for me and i could be doing something wrong. will have another bash at this tomorrow.

Reply:
FYI this works: `dokku docker-options:add app build "--no-cache --pull"`

Reply:
I've added some info to the README on how to handle the moving tag. Am happy to change it to use a commit sha if you'd prefer.

Reply:
I think I'd prefer using the commit sha. You can leave a comment in the workflow example showing how this might be different for a persistent tag.